### ISSUE TITLE
Cannot pick other species at roundstart when running a local server.

### ISSUE THREAD

**Issue by nero1024** [tgstation/tgstation#39145]
I'm running my own local test server built using the source .zip provided by the repository and I can't pick a species other than human despite whatever setting I am using in the game_options.txt file. Dream Daemon is throwing a runtime and it might be related to the issue I'm having, here is the server log from the daemon below:

```
Logging in...connected
Logged in as Nero1024.
Sat Jul 14 19:21:30 2018
World opened on network port 1280.
Welcome BYOND! (5.0 Beta Version 512.1438)
579 global variables
World loaded at 19:21:45!
## WARNING: /tg/station 13 uses many file operations, a few shell()s, and some external call()s. Trusted mode is recommended. You can download our source code for your own browsing and compilation at https://github.com/tgstation/tgstation 
Running /tg/ revision:
runtime error: bad index
proc name: Get (/datum/controller/configuration/proc/Get)
  source file: configuration.dm,178
  usr: null
  src: Configuration (/datum/controller/configuration)
  call stack:
Configuration (/datum/controller/configuration): Get(/datum/config_entry/number/err... (/datum/config_entry/number/error_msg_delay))
 (/datum/error_viewer/error_cache): log error(list index out of bounds (/exception), /list (/list), null)
world: Error(list index out of bounds (/exception), null)
/datum/getrev (/datum/getrev): New()
world: New()
[19:21:45] Runtime in getrev.dm,12: list index out of bounds
  proc name: New (/datum/getrev/New)
  src: /datum/getrev (/datum/getrev)
  call stack:
  /datum/getrev (/datum/getrev): New()
  world: New()
Loading config file maps.txt...
Unable to locate admins backup file.
Unable to locate admins backup file.
Initialized Database subsystem within 0 seconds!
Initialized Blackbox subsystem within 0 seconds!
Initialized Server Tasks subsystem within 0 seconds!
Initialized Input subsystem within 0 seconds!
Initialized Research subsystem within 0 seconds!
Initialized Events subsystem within 0.1 seconds!
Initialized Jobs subsystem within 0 seconds!
Initialized Quirks subsystem within 0 seconds!
Initialized Ticker subsystem within 0 seconds!
Loading Delta Station...
Ban database connection failure. Key nero1024 not checked
Ban database connection failure. Key nero1024 not checked
Ban database connection failure. Key nero1024 not checked
Ban database connection failure. Key nero1024 not checked
Ban database connection failure. Key nero1024 not checked

BYOND hub reports port 1280 cannot be reached by players.

A firewall or router may be interfering with your connection. For information on hosting worlds through a firewall or router, please visit BYOND Help or portforward.com.

Loaded Station in 61.8s!
Loaded Lavaland in 7.2s!
Ruin loader finished with 0 left to spend.
Ruin loader finished with 0 left to spend.
Initialized Mapping subsystem within 86.4 seconds!
Initialized Networks subsystem within 0 seconds!
Initialized Atoms subsystem within 50.3 seconds!
Initialized Language subsystem within 0 seconds!
Initialized Machines subsystem within 1.1 seconds!
Initialized Circuit subsystem within 0.1 seconds!
Initialized Disease subsystem within 0 seconds!
Initialized Medals subsystem within 0 seconds!
Initialized Night Shift subsystem within 0 seconds!
Initialized Parallax subsystem within 0 seconds!
Initialized Traumas subsystem within 0.3 seconds!
Initialized Weather subsystem within 0 seconds!
Initialized Atmospherics subsystem within 87.3 seconds!
Initialized Minimap subsystem within 0.2 seconds!
Initialized Assets subsystem within 2.9 seconds!
Initialized Icon Smoothing subsystem within 8.8 seconds!
Initialized Overlay subsystem within 1.4 seconds!
Initialized XKeyScore subsystem within 0 seconds!
Initialized Sticky Ban subsystem within 0.1 seconds!
Initialized Lighting subsystem within 14.6 seconds!
Initialized Shuttle subsystem within 3.6 seconds!
Initialized Squeak subsystem within 0.1 seconds!
Initialized Pathfinder subsystem within 0 seconds!
Initialized Persistence subsystem within 0.1 seconds!
Initializations complete within 257.6 seconds!
Game start took 0s
```
The only config settings I tweaked was lowering the LOBBY_COUNTDOWN time, disable ALLOW_RANDOM_EVENTS, enabled various ROUNDSTART_RACES, and enabled JOIN_WITH_MUTANT_HUMANS.

I'm using the latest BYOND beta 5.0 build 512.1438

[Round ID]: # N/A

[Testmerges]: # N/A

[Reproduction]: # Attempt to enter Setup Character and change species before joining round.

**Comment by SpaceManiac** [comment]
Fixed by #39126

**Comment by nero1024** [comment]
Can confirm, the fix works
